axelrod/tests/unit/test_finite_state_machines.py

```diff
@@ -0,0 +1,236 @@
+"""Tests for Finite State Machine Strategies."""
+import unittest
+
+import axelrod
+from .test_player import TestHeadsUp, TestPlayer
+
+C, D = axelrod.Actions.C, axelrod.Actions.D
+
```

Comment:
In addition to the four explicit tests? Right now you can't actually instantiate the FSMPlayer without supplying arguments (but we can pick a default of course).